src/participant/participant.ts

```diff
@@ -229,8 +231,15 @@ export default class ParticipantController {
     }
   }
 
-  // @MongoDB what is mongodb?
-  async handleGenericRequest(
+  _handleCancelledRequest({
+    context,
+  }: {
+    context: vscode.ChatContext;
+  }): ChatResult {
+    return createCancelledRequestChatResult(context.history);
+  }
+
+  async _handleRoutedGenericRequest(
```

Comment:
Moving this cancelled request handling and the other abort controller changes into a separate pr.